c/meson.build

```diff
@@ -144,15 +144,15 @@ nns_capi_pipeline_dep = declare_dependency(link_with: nns_capi_pipeline_lib)
 
```

Comment:
Your commit obsoletes the need for ```nns_capi_[single/pipeline]_lib```. Can you remove LINE 139-143, 115-119 and update other lines using them?